src/mpyl/steps/deploy/bpmn.py

```diff
@@ -0,0 +1,97 @@
+"""Deploys the Camunda diagrams in the build stage to Camunda cluster, using BPM. """
+import os
+from logging import Logger
+from python_on_whales import docker, Container, DockerException
+from ...utilities.docker import execute_with_stream
+from ...utilities.bpmn import CamundaConfig
+from . import STAGE_NAME
+from .. import Step, Meta
+from ..models import Input, Output, ArtifactType
+
+
+class BpmnDiagramDeploy(Step):
+    def __init__(self, logger: Logger) -> None:
+        super().__init__(
+            logger,
+            Meta(
+                name="BPMN Diagram Deploy",
+                description="Deploy BPMN diagram to Camunda Cluster",
+                version="0.0.1",
+                stage=STAGE_NAME,
+            ),
+            produced_artifact=ArtifactType.NONE,
+            required_artifact=ArtifactType.NONE,
+            # TO DO: make an artifcatype with a camunda deployed diagram link
```

Comment:
this is too much work to do now or was it forgotten?

Reply:
I remark it because it is not the work for now. It's not required to build our cicd pipeline but it's good to have. Also because I am going on holiday soon, this remark is to remind my teammates (or maybe me) to pick it up afterwards. A ticket is also created for this